api/src/pdc/services/policy/engine/helpers/dateRange.ts

```
@@ -0,0 +1,40 @@
+ * @param r
+ * @returns
+ */
+export function dateRange(...r: Array<Date | string>): string[] {
+  if (r.length === 0) {
+    throw new Error("At least one date is required");
+  }
+
+  const range = r
+    .map((d) => {
+      const date = typeof d === "string" ? new Date(d) : d;
+      if (isNaN(date.getTime())) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)